lib/state/revisions/actions.js

```diff
@@ -0,0 +1,3 @@
+export const setRevisionState = revisionState => ( {
+	type: 'setRevisionState', revisionState,
```

Comment:
you might want to checkout out `state/action-types.js` where we can declare global action types.

warning though, in **master** I have code there that I put in but I don't like. instead of defining action types with `Symbol()` I think I would like to stick with uppercase snake-case strings so that tooling is easier